apps/geomat/src/assets/data/guidedTour.json

```diff
@@ -90,5 +90,10 @@
       "element": "#profile",
       "intro": "Wir wünschen dir viel Spaß beim Erkunden und Bestimmen verschiedener Minerale."
     }
+  ],
+  "location": [
```

Comment:
Gibt es einen Grund, warum `location` ein Array sein muss?
Es könnte doch auch ein object oder sogar direkt ein string (= die URL) sein?

Reply:
Also weil ich gedacht habe, dass wir vielleicht in der Zukunft mehrere URL in dem `location` haben wollen, dann brauchen wir wahrscheinlich noch ein key-value Paar um die URL zu identifizieren. Aber brauchen wir das nicht oder =.=

Reply:
Hmmm... wenn ich  mir das so anschauen... Wäre es nicht eigentlich logisch, `location` als Teil der jeweiligen Sub-Tour zu haben? Ich meine so:
```
{
  "landing": {
    "location": "url",
    "steps": [
      {
        "intro": "..."
      },
      {
        "..."
      }
    ]
  }
}
```

Wäre das möglich?

Reply:
> Hmmm... wenn ich mir das so anschaue... Wäre es nicht eigentlich logisch, `location` als Teil der jeweiligen Sub-Tour zu haben? Ich meine so:
> 
> ```
> {
>   "landing": {
>     "location": "url",
>     "steps": [
>       {
>         "intro": "..."
>       },
>       {
>         "..."
>       }
>     ]
>   }
> }
> ```
> 
> Wäre das möglich?

Yup. Das ist doch besser

Reply:
Wie ist das bei der profile-Tour am Ende? Am Ende sollten wir ja wieder auf der allgemeinen Sicht landen, ohne, dass ein Steckbrief geöffnet ist. Brauchen wir da nicht schon zwei URLs? Können wir die url nicht sogar in den `step` einfügen?

Reply:
Das muss ich kurz ausprobieren, weil wenn wir nach `profile/tree` navigieren, ist der Tree noch angezeigt. Es ist wahrscheinlich nicht gewünscht.
![Screenshot (56)](https://user-images.githubusercontent.com/33594579/160848361-974b3e13-c13b-42b5-a3c9-2a9d823d4681.png)

Reply:
Ja, es wäre besser, der Tree wäre wieder zu, aber problematisch ist das auch nicht. 
Der Tree wird nicht durch die `navigation` geöffnet, das passiert anders (wie, weiß ich gerade nicht).